src/python/zquantum/qaoa/estimators.py

```diff
@@ -20,18 +21,16 @@ class CvarEstimator(Estimator):
     "Improving Variational Quantum Optimization using CVaR", P. Barkoutsos, G. Nannicini, A. Robert, I. Tavernelli, and S. Woerner
     """
 
-    @overrides
-    def get_estimated_expectation_values(
-        self,
-        backend: QuantumBackend,
-        circuit: Circuit,
-        target_operator: IsingOperator,
-        alpha: float,
-        n_samples: Optional[int] = None,
-    ) -> ExpectationValues:
+    def __init__(self, alpha: float) -> None:
```

Comment:
`alpha` should be described in the docstring, especially since it's an impactful parameter.

Reply:
See: https://github.com/zapatacomputing/z-quantum-qaoa/pull/44/commits/58acb9946120ecbe079898018c01eec06cc6d596